Documentation/gettingstarted/encryption.rst

```diff
@@ -107,6 +107,11 @@ In order to enable node-to-node encryption, add:
     --set encryption.enabled=true \
     --set encryption.nodeEncryption=true
 
+.. note::
+
+    Node to node encryption feature is tested and supported with direct routing
+    modes. Using with encapsulation/tunneling is not currently tested or supported.
```

Comment:
:+1: We could add a link to the related GH issue here as well so at least when someone reads this, they can click through the link to find out if this case was addressed, report they're trying it out, or want to work on it for instance.